--- ./configure.ac	2022-08-04 09:40:19.663785551 +0200
+++ ../../tmp/systemtap/configure.ac	2022-08-04 09:22:31.531742996 +0200
@@ -546,7 +546,8 @@
       [AC_SUBST(preferred_python,[$PYTHON3])],
       [test "x$PYTHON" != "x:"],
       [AC_SUBST(preferred_python,[$PYTHON])],
-      [AC_MSG_ERROR([neither python version 2 nor 3 found])])
+      #[AC_MSG_ERROR([neither python version 2 nor 3 found])]
+      )
       
 if test $enable_translator = yes; then
     # Need libdwfl-capable recent elfutils http://elfutils.org/
@@ -574,8 +575,8 @@
 # staprun has more modest libelf needs
 save_LIBS="$LIBS"
 dnl this will only succeed with elfutils 0.142+
-AC_CHECK_LIB(elf,elf_getshdrstrndx,[],[
-  AC_MSG_FAILURE([libelf too old, need 0.142+])])
+#AC_CHECK_LIB(elf,elf_getshdrstrndx,[],[
+#  AC_MSG_FAILURE([libelf too old, need 0.142+])])
 staprun_LIBS="$staprun_LIBS -lelf"
 stapbpf_LIBS="$stapbpf_LIBS -lelf"
 LIBS="$save_LIBS"
